data/tutorials/gs_00_up_and_running.md

```diff
@@ -166,9 +165,14 @@ Now that the tools are installed, it remains to understand how to use them. Most
 
 ## Using the OCaml Toplevel with UTop
 
-UTop is a nice toplevel for OCaml. Unlike the standard top-level with the `ocaml` command, UTop features history, tab completion, line edition, and the ability to load a package installed in your switch.
+UTop is an extended and improved toplevel (REPL) for OCaml. Unlike the standard toplevel with the `ocaml` command, UTop features history, tab completion, interactive line editing, and the ability to load any package installed in your switch.
 
-By running the `utop` command, it will print out something similar to what you see below. After the horizontal line, you will see a new prompt `utop #`. Here is where you type your expressions. In this case `1 + 2 * 3;;`. After you press Enter, the following line shown is the desired output. 
+If you have never used a toplevel (REPL) before: think of it as an interactive
+terminal/shell that evaluates expressions. You type an OCaml expression, then
+press the Enter key or the Return key, and the toplevel responds with the value
+of the evaluated expression. 
+
+Running the `utop` command from your shell, it will print out something similar to what you see below. Below the horizontal line, you find the prompt `utop #`. Here is where you type your expressions.
```

Comment:
```suggestion
When running the `utop` command from your shell, it will print out something similar to what you see below. Below the horizontal line, you find the prompt `utop #`. Here is where you type your expressions.
```